components/toggle/toggle.vue

```diff
@@ -138,6 +150,10 @@ export default {
         },
       ];
     },
+
+    shouldShowIcon () {
+      return this.showIcon;
+    },
```

Comment:
Can't we just use the showIcon prop?

Reply:
Yeah for this PR should be enough. But the next feature is to add the indeterminate state which shouldn't show the icon and so this computed prop logic will be modified to something like:

```js
  shouldShowIcon () {
    return !this.indeterminate && this.showIcon;
  },
```

Reply:
update: you're right, no need to use a computed prop since the icons are hidden in the indeterminate state [with the CSS classes](https://github.com/dialpad/dialtone/blob/staging/lib/build/less/components/toggle.less#L132-L134). Removed https://github.com/dialpad/dialtone-vue/pull/573/commits/497e0487456bf7820caebf5f4e6324b361db2990